src/lib/offline-api/index.ts

```diff
@@ -570,6 +583,26 @@ class OfflineAPI {
     return tag
   }
 
+  async deleteTag (id: string) {
+    this.assertRecording()
+
+    const tag = await this.getTag(id)
+    // Store clone as a request
+    this.currentRequestsRecorded.push(deleteTagRequest(tag.clone()))
+
+    this.modifiedTags.delete(id)
+    this.savedTags.delete(id)
+
+    // TODO Do we need this in the case of tags? What does it validate
```

Comment:
I think we can skip it. 
Not sure if we even have a hook for `DeleteContentType`, but I can imagine we would have wanted to validate that there are no existing entries for that CT in case of a CT deletion. Otherwise the API will reject that deletion.
For tags we have nothing that needs to be validated AFAIK so I think it can be skipped.